### PR TITLE
Migrate yarn

### PR DESCRIPTION
**Pull request by wwalser**
chore: implement steps necessary to move local commands and build system to modern yarn

### CODE REVIEW

File: packages/hooks/src/useTracking/index.test.tsx [hunk 1]
```diff
@@ -1,9 +1,9 @@
-import { FieldDictionary, Pipeline } from '@sajari/react-hooks';
+// import { FieldDictionary, Pipeline } from '@sajari/react-hooks';
```
Comment (JasonBerry) [inline packages/hooks/src/useTracking/index.test.tsx:1]: Remove

**Review by JasonBerry — APPROVED** [review]
When we update yarn next time is there a simple command that will update the files we've committed to our repo under `.yarn` or is that all pretty manual?

**Comment by wwalser** [comment]
> When we update yarn next time is there a simple command that will update the files we've committed to our repo under .yarn or is that all pretty manual?

`yarn set version *whatever*`

**Comment by wwalser** [comment]
Builds and tests are running now but I have a single test failure that I can't reproduce locally.

**Comment by wwalser** [comment]
@zlatanpham I've hit my head on this for a while, can you investigate why that test is failing? 
Also happy for @tuanddd to have a look as I think he's the original author of the test. I'd be curious to know if it's failed in this manner elsewhere.

**Comment by JasonBerry** [comment]
We're still using node 12.x in the release.yml (instead of the version we specify in nvm) and run-test.yml doesn't even specify a version… who knows what it's using. That could be a potential source of the failure.

**Comment by wwalser** [comment]
Yup, I'll switch to using the nvmrc file.

**Comment by wwalser** [comment]
-_-

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -20,13 +20,31 @@ jobs:
           fetch-depth: 0
           persist-credentials: false
 
-      - name: Use Node.js 12.x
-        uses: actions/setup-node@v1
+      - name: Setup node
+        uses: actions/setup-node@v2
         with:
-          version: 12.x
+          node-version-file: '.nvmrc'
```
Comment (JasonBerry) [inline .github/workflows/release.yml:26]: TIL: `node-version-file` is a thing. I now feel the need to fix all the actions into which I introduced that bash script to manually use nvm.

**Review by JasonBerry — APPROVED** [review]

**Pull request merged**